[2022-04-28 05:21:58,314] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.format_to_parquet_task scheduled__1998-01-01T00:00:00+00:00 [queued]>
[2022-04-28 05:21:58,363] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.format_to_parquet_task scheduled__1998-01-01T00:00:00+00:00 [queued]>
[2022-04-28 05:21:58,366] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-28 05:21:58,368] {taskinstance.py:1250} INFO - Starting attempt 1 of 2
[2022-04-28 05:21:58,369] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-28 05:21:58,420] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 1998-01-01 00:00:00+00:00
[2022-04-28 05:21:58,430] {standard_task_runner.py:52} INFO - Started process 4823 to run task
[2022-04-28 05:21:58,439] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcp_dag', 'format_to_parquet_task', 'scheduled__1998-01-01T00:00:00+00:00', '--job-id', '434', '--raw', '--subdir', 'DAGS_FOLDER/ingest_to_gcs_dag.py', '--cfg-path', '/tmp/tmpvbrwtvup', '--error-file', '/tmp/tmprngl1e2m']
[2022-04-28 05:21:58,442] {standard_task_runner.py:80} INFO - Job 434: Subtask format_to_parquet_task
[2022-04-28 05:21:58,884] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcp_dag.format_to_parquet_task scheduled__1998-01-01T00:00:00+00:00 [running]> on host 65f5b8b3c17c
[2022-04-28 05:21:59,202] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-28 05:21:59,389] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcp_dag
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=1998-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__1998-01-01T00:00:00+00:00
[2022-04-28 05:21:59,402] {taskinstance.py:1774} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/ingest_to_gcs_dag.py", line 65, in format_to_parquet
    table = pcsv.read_csv(src_file)
  File "pyarrow/_csv.pyx", line 881, in pyarrow._csv.read_csv
  File "pyarrow/_csv.pyx", line 774, in pyarrow._csv._get_reader
  File "pyarrow/io.pxi", line 1631, in pyarrow.lib.get_input_stream
  File "pyarrow/io.pxi", line 1602, in pyarrow.lib.get_native_file
  File "pyarrow/io.pxi", line 896, in pyarrow.lib.OSFile.__cinit__
  File "pyarrow/io.pxi", line 906, in pyarrow.lib.OSFile._open_readable
  File "pyarrow/error.pxi", line 143, in pyarrow.lib.pyarrow_internal_check_status
  File "pyarrow/error.pxi", line 112, in pyarrow.lib.check_status
FileNotFoundError: [Errno 2] Failed to open local file '/opt/airflow/us_employment_98.csv'. Detail: [errno 2] No such file or directory
[2022-04-28 05:21:59,513] {taskinstance.py:1288} INFO - Marking task as UP_FOR_RETRY. dag_id=data_ingestion_gcp_dag, task_id=format_to_parquet_task, execution_date=19980101T000000, start_date=20220428T052158, end_date=20220428T052159
[2022-04-28 05:21:59,567] {standard_task_runner.py:98} ERROR - Failed to execute job 434 for task format_to_parquet_task ([Errno 2] Failed to open local file '/opt/airflow/us_employment_98.csv'. Detail: [errno 2] No such file or directory; 4823)
[2022-04-28 05:21:59,618] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-04-28 05:21:59,887] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-28 06:04:13,806] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.format_to_parquet_task scheduled__1998-01-01T00:00:00+00:00 [queued]>
[2022-04-28 06:04:13,866] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.format_to_parquet_task scheduled__1998-01-01T00:00:00+00:00 [queued]>
[2022-04-28 06:04:13,868] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-28 06:04:13,869] {taskinstance.py:1250} INFO - Starting attempt 1 of 2
[2022-04-28 06:04:13,871] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-28 06:04:13,925] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 1998-01-01 00:00:00+00:00
[2022-04-28 06:04:13,937] {standard_task_runner.py:52} INFO - Started process 1554 to run task
[2022-04-28 06:04:13,946] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcp_dag', 'format_to_parquet_task', 'scheduled__1998-01-01T00:00:00+00:00', '--job-id', '516', '--raw', '--subdir', 'DAGS_FOLDER/ingest_to_gcs_dag.py', '--cfg-path', '/tmp/tmpfrks8dkv', '--error-file', '/tmp/tmpvz6ys6cs']
[2022-04-28 06:04:13,949] {standard_task_runner.py:80} INFO - Job 516: Subtask format_to_parquet_task
[2022-04-28 06:04:14,362] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcp_dag.format_to_parquet_task scheduled__1998-01-01T00:00:00+00:00 [running]> on host d4721d223bdd
[2022-04-28 06:04:14,647] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-28 06:04:14,843] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcp_dag
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=1998-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__1998-01-01T00:00:00+00:00
[2022-04-28 06:04:15,301] {python.py:175} INFO - Done. Returned value was: None
[2022-04-28 06:04:15,430] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcp_dag, task_id=format_to_parquet_task, execution_date=19980101T000000, start_date=20220428T060413, end_date=20220428T060415
[2022-04-28 06:04:15,536] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-28 06:04:16,181] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
